code/game/objects/items/powerfist.dm

```diff
@@ -19,6 +19,7 @@
 	var/gasperfist = 3
 	var/obj/item/tank/internals/tank = null //Tank used for the gauntlet's piston-ram.
 	var/baseforce = 20
```

Comment:
Reduce the baseforce to 10.
This maintains balance with the e-sword at 30 damage per hit at tier 3,

Remember, a tier 1 freezer setup can get you 30 hits in a single air tank.
This, plus the FIFTEEN tile knockback makes it incredibly strong at 70 damage a hit.

Reply:
this would deal half 2.5 times less damage than the esword at tier 3 as it attacks 2.5x slower